src/ActivitiesSDKAdapter.js

```diff
@@ -18,5 +22,66 @@ import {ActivitiesAdapter} from '@webex/component-adapter-interfaces';
 export default class ActivitiesSDKAdapter extends ActivitiesAdapter {
   constructor(datasource) {
     super(datasource);
+
+    this.activityObservables = {};
+  }
+
+  /**
+   * Returns a promise to activity data from Webex.
```

Comment:
Loads activity data from Webex and returns a promise that resolves to an Activity object